k8s/secret.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgres-creds
+data:
+  database-uri: cG9zdGdyZXNxbCtwc3ljb3BnOi8vcG9zdGdyZXM6cGdzM2NyM3RAcG9zdGdyZXM6NTQzMi9wb3N0Z3Jlcw==
```

Comment:
this one should be exactly the same as we used in `deployment.yaml`. you use `database_uri` in `deployment.yaml`. you can either choose `_` or `-`. I'm fine with both.

Reply:
Thanks for catching that! I just changed it to database-uri in deployment.yaml